.github/workflows/call-run-tests.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: npx playwright install --with-deps
 
       - name: Run Playwright tests
-        run: npm run gh:test
+        run: npm run test
```

Comment:
@dualcnhq I thought you mentioned that we are clearing these workflow changes from CR?

Reply:
yes, but this specific workflow is also used in the main workflow.

the `npm run gh:test` has a `--trace on` parameter which takes a lot of memory on GH web runner so switching it back for now to the usual run test command